BurnOutSharp/ProtectionType/SafeDisc.cs

```diff
@@ -83,28 +78,10 @@ public string CheckPortableExecutable(string file, PortableExecutable pex, bool
             if (stxt371Section || stxt774Section)
                 return $"SafeDisc {Get320to4xVersion(null, null, null)}";
 
-            // Get the BSS section, if it exists
-            var BSSSectionRaw = pex.ReadRawSection("BSS", first: true);
-            if (BSSSectionRaw != null)
-            {
-                var matchers = new List<ContentMatchSet>
-                {
-                    // Present in "mcp.dll", along with several other references to Plextor (Redump entries 28810 and 30555).
-                    // plextor.dll.IsPlextor.PreventLowSpeed
-                    new ContentMatchSet(new byte?[]
-                    {
-                        0x70, 0x6C, 0x65, 0x78, 0x74, 0x6F, 0x72, 0x2E, 
-                        0x64, 0x6C, 0x6C, 0x00, 0x49, 0x73, 0x50, 0x6C, 
-                        0x65, 0x78, 0x74, 0x6F, 0x72, 0x00, 0x50, 0x72,
-                        0x65, 0x76, 0x65, 0x6E, 0x74, 0x4C, 0x6F, 0x77, 
-                        0x53, 0x70, 0x65, 0x65, 0x64
-                    }, "SafeDisc (Version 1.45.011-1.50.020)"),
-                };
-
-                string match1 = MatchUtil.GetFirstMatch(file, BSSSectionRaw, matchers, includeDebug);
-                if (!string.IsNullOrWhiteSpace(match1))
-                    return match1;
-            }
+            // Present on all "CLOKSPL.EXE" versions before SafeDisc 1.06.000. Found on Redump entries 61731 and 66004. 
+            // Only found so far on SafeDisc 1.00.025-1.01.044, but the report is currently left generic due to the generic nature of the check.
+            if (!string.IsNullOrWhiteSpace(name) && name.Equals("SafeDisc", StringComparison.OrdinalIgnoreCase))
```

Comment:
Since you moved this, make sure that you either make a note that this is from the file description or grab it again right before.